src/atoms/InputLabel/InputLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { FiHelpCircle } from "react-icons/fi";
+import { ICustomLabelProps } from "./InputLabel.model";
+import "./inputLabel.scss";
+
+
+const Label = ({
+  label,
+  toolTip,
+  labelType,
+}: React.PropsWithChildren<ICustomLabelProps>): React.ReactElement => {
+  return (
+    <label className="label">
```

Comment:
Using the `label` tag here instead of just a `div` is good, semantic HTML is important in React too 🎉